jobs/bbr-smbbroker/templates/post-restore-unlock.sh.erb

```diff
@@ -3,8 +3,6 @@
 set -euo pipefail
 set -x
 
-PATH="/var/vcap/packages/cf-cli-6-linux/bin:${PATH}"
-PATH="/var/vcap/packages/cf-cli-7-linux/bin:${PATH}"
```

Comment:
@davewalter It looks cf-cli-7 is still [supported](https://github.com/cloudfoundry/bosh-package-cf-cli-release/tree/main/packages). Our CI is failing because that's what it was using. Can we only remove cf-cli-6 and keep the other two for now?

Reply:
I've restored the cf-cli-7-linux package addition to the PATH so it should be able to run now.